blueoil/utils/tfds_builders/object_detection.py

```diff
@@ -77,8 +81,8 @@ def _generate_examples(self, dataset):
                     "bbox": tfds.features.BBox(
                         ymin / height,
                         xmin / width,
-                        (ymin + h) / height,
-                        (xmin + w) / width,
+                        _clip_max((ymin + h) / height),
```

Comment:
@hadusam 
`min((ymin + h) / height, 1.0)` is not enough for you?

Reply:
@iizukak 
Yeah, it's enough! Thanks!!!
Fixed.